app/jobs/user/statsd_gauge_online_user_count_job.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-module Statsd
-  class GaugeOnlineUserCountJob < ApplicationJob
+class User
+  class StatsdGaugeOnlineUserCountJob < ApplicationJob
```

Comment:
not a great name, but i'm getting this in for the sake of time since sidekiq jobs aren't running due to this error